Library/Homebrew/manpages/brew-cask.1.md

```diff
@@ -86,6 +86,12 @@ names, and other aspects of this manual are still subject to change.
     If <token> is given, summarize the staged files associated with the
     given Cask.
 
+  * `reinstall` <token> [ <token> ...]
+    Reinstall the given Cask.
+
+    <token> is usually the ID of a Cask as returned by `brew cask search`,
+    but see [OTHER WAYS TO SPECIFY A CASK][] for variations.
```

Comment:
I don't think we need to repeat what a `token` is here. Otherwise we would need to repeat if for every command that takes a `token`.